site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,233 @@
+          <li class="nav-item"><a class="nav-link" href="#">Data only cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Top up</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Assistance</a></li>
+        </ul>
+        <ul class="navbar-nav border-top border-1 border-dark d-flex flex-row d-lg-none">
+          <li class="nav-item"><a class="nav-link active" href="#" aria-label="EN English version" aria-current="true">EN</a></li>
+          <li class="nav-item"><a class="nav-link" href="#" aria-label="FR Version française" lang="fr" hreflang="fr">FR</a></li>
+        </ul>
+      </div>
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#user"></use>
+              </svg>
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<main class="bd-content order-1">
+  <div class="bg-white title-bar">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-sm-12 col-lg-5 offset-lg-3">
+          <h1 class="display-1">Become a reseller</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-4">
+    <div class="row">
+      <div class="col-sm-12 col-lg-5 offset-lg-3">
+        <p class="ll-sm h3 mb-4">You are a professional and wish to resell Orange products to your
+          customers? Please fill the form and we will get back to you shortly.
+        </p>
+        <form class="needs-validation mb-5" novalidate>
+          <div class="row">
+            <div class="col-12">
+              <p class="fw-bold mb-4">
+                <span class="text-primary">*</span> indicates required
+              </p>
+            </div>
+            <div class="col-sm-4 col-md-6 mb-3">
+              <label for="selectTitle" id="selectTitleLabel" class="form-label is-required">Title</label>
+              <select class="form-select" id="selectTitle" aria-labelledby="selectTitleLabel selectTitleFeedback"
```

Comment:
OK because this is in the design guidelines but I don't think this is a really inclusive approach to select gender identity. Let's make this feedback to the designers.

Reply:
Actually nothing's defined in https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53. So we could add more inclusive options. Let me try to find something.